app/models/concerns/findings/csv.rb

```diff
@@ -76,12 +88,24 @@ def issue_date_text
       issue_date ? I18n.l(issue_date, format: :minimal) : '-'
     end
 
+    def final_created_at_text
+      review.conclusion_final_review ? I18n.l(review.conclusion_final_review.created_at, format: :minimal) : '-'
+    end
+
     def date_text
       date = solution_date || follow_up_date
 
       date ? I18n.l(date, format: :minimal) : '-'
     end
 
+    def taggings_format
```

Comment:
```suggestion
    def taggings_format
	tags = taggings.map(&:tag)    
	
	USE_SCOPE_CYCLE ? tags.join ' - ' : tags.to_sentence
    end
```

Este si te gusta, es para no repetir el codigo

Reply:
Muchas gracias